apps/web/src/components/layout/components/v2/BridgeUpdateModalTrigger.tsx

```diff
@@ -32,16 +32,9 @@ function BridgeUpdateModalTriggerControl({ onClick }: { onClick: () => void }) {
   const { status, bridgeURL } = useBridgeConnectionStatus();
 
   const trigger = isHovered ? (
-    <button
-      {...hoverProps}
-      onClick={onClick}
-      className={css({ '&, & svg': { color: 'typography.text.main !important' } })}
-    >
-      <HStack gap="25">
-        <IconEdit className={css({ color: 'icon.main' })} size="16" />
-        <Text>Edit endpoint</Text>
-      </HStack>
-    </button>
+    <Button {...hoverProps} variant="outline" size="xs" Icon={IconEdit} onClick={onClick}>
+      <Text>Edit endpoint</Text>
```

Comment:
```suggestion
      Edit endpoint
```
The Button component will handle the design of the text :)

Reply:
Nice catch!